.github/workflows/python-package.yml

```diff
@@ -29,3 +35,27 @@ jobs:
         run: |
           # Run tox using the version of Python in `PATH`
           tox -e py
+
+  build-windows:
+
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9, '3.10.0-beta.1']
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pytest tox
+          if exist requirements.txt pip install -r requirements.txt
```

Comment:
I figured out how to get the github windows CI to trigger and committed that config update to your branch.  it's erroring on this line, perhaps it runs under a different shell?

Reply:
Ah, it's using powershell. I pushed an update.